pkg/proctree/tree/tree.go

```diff
@@ -0,0 +1,190 @@
+package tree
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"sync"
+
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/aquasecurity/tracee/pkg/proctree/nodes"
+)
+
+type Tree struct {
+	// TODO processes map will be a LRU map with eviction by cache limit and time
+	processes map[uint32]*nodes.Process // map[murmur3 hash of process] -> process node
```

Comment:
BTW I recently read that go actually does extra GC scans if pointers are included in a map's definition (https://go101.org/optimizations/6-map.html#:~:text=16.10%20ns/op-,Pointers%20in%20maps,-If%20the%20key).
Considering this map will get large, we should probably do a regular struct here.

Reply:
This actually relates to a lot of code we have, so we may consider opening an issue about this.

Reply:
This is an optimization that I believe we're very far away to achieve. There is no real direct action right now I believe (so marking this as solved).